future/src/ct/ct_widgets.cc

```diff
@@ -616,19 +617,6 @@ void CtTextView::for_event_after_key_press(GdkEvent* event, const Glib::ustring&
                     if (iter_start.get_line_offset() == 0 and iter_start.get_char() == g_utf8_get_char(CtConst::CHAR_COLON))
                         // ":: " becoming "▪ " at line start
                         _special_char_replace(CtConst::CHARS_LISTBUL_DEFAULT[2], iter_start, iter_insert);
-                } else if (_pCtMainWin->get_ct_config()->enableMdFormatting && syntaxHighlighting == CtConst::RICH_TEXT_ID) {
```

Comment:
I wonder, you decided to remove real time formatting?

Reply:
Remove it after space, it still activates on enter (As I mentioned in the description)

Reply:
It should possibly be a config option, but I found when using it that space was very annoying when trying to edit stuff

Reply:
Ah, I see, for some reason, I thought removing formatting meant converting)